lua/nvim-tree.lua

```diff
@@ -255,9 +255,9 @@ function M.on_enter(netrw_disabled)
   end
 
   local should_hijack = _config.hijack_directories.enable
-    and _config.hijack_directories.auto_open
```

Comment:
Unrelated, please revert that hunk, it makes for cleaner history.

Reply:
Done

Reply:
Thanks!